packages/nextjs/app/builders/0x4b2b0D5eE2857fF41B40e3820cDfAc8A9cA60d9f/page.tsx

```diff
@@ -0,0 +1,103 @@
+"use client";
+
+import Link from "next/link";
+import type { NextPage } from "next";
+import { BellIcon, BriefcaseIcon, CodeBracketIcon, FaceSmileIcon } from "@heroicons/react/24/outline";
+
+const Wilman: NextPage = () => {
+  return (
+    <>
+      <div className="flex items-center flex-col flex-grow pt-10">
+        <div className="px-5">
+          <h1 className="text-center">
+            <span className="block text-2xl mb-2">Hello, Batch 6!</span>
+            <span className="block text-4xl font-bold">I am Wilman Vinueza</span>
+          </h1>
+          <p className="text-center text-lg mt-4">Blockchain Developer & Solidity Enthusiast </p>
+        </div>
+
+        <div className="px-5 py-10 bg-base-300">
+          <h2 className="text-2xl font-bold text-center mb-4">About Me</h2>
+          <div className="max-w-xl mx-auto">
+            <p className="text-center text-lg mb-4">
+              I am an engineer based in Ecuador. For the past few years I have been learning all things blockchain,
+              ethereum and solidity.
+            </p>
+            <p className="text-center text-lg">
+              When I am not coding, you can find me tinkering with electronics, solving puzzles, swimming or playing
+              ping pong.
+            </p>
+          </div>
+        </div>
+
+        <div className="flex-grow w-full mt-16 px-8 py-12">
+          <div className="flex justify-center items-center gap-8 flex-col sm:flex-row">
+            <div className="flex flex-col bg-base-100 px-6 py-6 text-center items-center max-w-xs rounded-3xl">
+              <CodeBracketIcon className="h-8 w-8 fill-secondary" />
+              <p>
+                Explore my projects on{" "}
+                <Link href="https://github.com/wildanvin" passHref className="link">
```

Comment:
I'd be better if the links are target="_blank". 

You can also use `<a>` directly, since <Link> is meant for internal routes (prefeching, etc)

Reply:
I understand, I will make those changes and push again